CP04OSSM/CP04OSSM_R00002_ingest.csv

```diff
@@ -1,45 +1,45 @@
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/instruments/dcl26/VELPT_sn11377/*.aqd,CP04OSSM-RID26-04-VELPTA000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl26/phsen1/*.phsen1.log,CP04OSSM-RID26-06-PHSEND000,recovered_host,Available,
+#mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/instruments/dcl26/PHSEN*/SAMI*.txt,CP04OSSM-RID26-06-PHSEND000,recovered_inst,Not Expected,"no SAMI file recovered. Only dcl, host data will have to suffice"
+mi.dataset.driver.nutnr_b.dcl_full.nutnr_b_dcl_full_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl26/nutnr/*.nutnr.log,CP04OSSM-RID26-07-NUTNRB000,recovered_host,Available,
+#mi.dataset.driver.nutnr_b.nutnr_b_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/instruments/dcl26/NUTNR_sn261/SCH*.DAT,CP04OSSM-RID26-07-NUTNRB000,recovered_inst,Not Expected,only host data available
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl26/spkir/*.spkir.log,CP04OSSM-RID26-08-SPKIRB000,recovered_host,Available,
 #mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/syslog/*.syslog.log,CP04OSSM-RID27-00-DCLENG000,recovered_host,Not Expected,no recovered ENG files
-mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/OPTAA_snACS240/*.optaa1.log,CP04OSSM-RID27-01-OPTAAD000,recovered_host,Available,
-mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/FLORT_sn994/*.flort.log,CP04OSSM-RID27-02-FLORTD000,recovered_host,Available,
-mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/CTDBP_sn16P71174-7208/*.ctdbp1.log,CP04OSSM-RID27-03-CTDBPC000,recovered_host,Available,
-mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/CTDBP_sn16P71174-7208/SBE*.hex,CP04OSSM-RID27-03-CTDBPC000,recovered_inst,Available,
-mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/DOSTA_sn410/*dosta1.log,CP04OSSM-RID27-04-DOSTAD000,recovered_host,Available,
-#mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/irid2shore/cpm_status*.txt,CP04OSSM-SBC11-00-CPMENG000,recovered_host,Not Expected,no recovered ENG files
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/optaa1/*.optaa1.log,CP04OSSM-RID27-01-OPTAAD000,recovered_host,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/flort1/*.flort.log,CP04OSSM-RID27-02-FLORTD000,recovered_host,Available,

```

Comment:
I don't believe you need the '1' on line 32 - flort1 I think should just be flort?

Reply:
It is flort1 in the raw data repo folder, so it looks correct here.

[flort1 on raw data repo](https://rawdata.oceanobservatories.org/files/CP04OSSM/R00002/cg_data/dcl27/)